src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccount/GetGovernmentFunding.cshtml

```diff
@@ -11,11 +11,15 @@
 
 <div class="grid-row">
     <div class="column-two-thirds">
-        <h1 class="heading-xlarge">
-            Get government funding
+        <h1 class="heading-xlarge">
+            Get government funding
```

Comment:
what is the change here? looks identical

Reply:
There isn't one.. Why has it highlighted that?